### DISCUSSION TITLE
october sprint: custom indexes

### DISCUSSION THREAD

**Discussion by keewis** [pydata/xarray#7084] [Announcements]
In the bi-weekly meetings we've been talking about having another sprint on custom indexes sometime in October, this time with as many interested core devs as possible.

To organize that, I think we'd need (feel free to edit this list):
- [ ] a time slot / day, with the only restriction being that @benbovy should be available for at least a part of the sprint
- [ ] a place to meet (the discord server from the scipy22 sprint?)
- [ ] some PRs / fixed issues:
  - [ ] #6975
  - [x] #6971
  - [ ] #6795
- [ ] maybe a list of other issues that may benefit from synchronous communication?
  - [ ] #6293 (any item someone would like to tackle)
  - [ ] https://github.com/pydata/xarray/discussions/7041#discussion-4394875

where as far as I can tell the only really urgent item is finding a time slot

**Comment by TomNicholas** [comment]
Sounds good, I would make a when2meet then tag the xarray team to get responses.

**Reply by keewis** [reply]
thanks, that would be great!

**Reply by TomNicholas** [reply]
I meant you should make the when2meet :sweat_smile: But I can also do it.

Is this a different hack session to the AMS one that some people were talking about with @dcherian?

**Reply by keewis** [reply]
oh, it seems I misunderstood your reply: I thought you were volunteering :sweat_smile:

Not sure, I don't remember hearing about that one. When would that be?

**Reply by dcherian** [reply]
That's this one: https://discourse.pangeo.io/t/pangeo-hackathon-at-ams-2023/2665

But yes, it should be different.

**Reply by keewis** [reply]
did we say what time frame we were aiming at? Do we use a full day? If so, what do we do about time zones? Do earlier time zones just start / end earlier?

Anyways, it would probably be good if someone else would create the poll: I tried yesterday, but I don't have any experience with doing such a thing for multiple time zones.
